### PR TITLE
Focus test

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Ability to run individual tests

**Does this PR change a user-facing CRD or CLI?**:

**Is a release note needed?**:

**Does this change need to be cherry-picked to a release branch?**:

### CODE REVIEW

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -98,30 +102,77 @@ func setup(t *testing.T) {
...
+	SetupStork(t)
+	t.Run("Mocktest-2", testMock)
+}
+
+func TestMock3(t *testing.T) {
+	SetupStork(t)
+	t.Run("Migration", testMigration)
+}
+
 func TestMain(t *testing.T) {
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: TestMain is no longer required.

Can the setup be done from init() instead? Don't need to change this function then, just need to remove the call to setup from here

Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Moved setup to init() function.

File: test/integration_test/common_test.go [hunk 2]
```diff
@@ -68,60 +68,89 @@ var snapshotScaleCount int
...
-func setup(t *testing.T) {
+func setup() error {
 	var err error
 
-	storkVolumeDriver, err = storkdriver.Get(volumeDriverName)
-	require.NoError(t, err, "Error getting stork driver %v", volumeDriverName)
+	if storkVolumeDriver, err = storkdriver.Get(volumeDriverName); err != nil {
+		return fmt.Errorf("Error getting stork driver %v: %v", volumeDriverName, err)
+	}
+	//require.NoError(t, err, "Error getting stork driver %v", volumeDriverName)
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: Remove commented code

**Comment by disrani-px** [comment]
Please squash the changes into one commit before merging.

File: test/integration_test/common_test.go (hunk 2 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

File: test/integration_test/common_test.go [hunk 3]
```diff
@@ -68,60 +68,87 @@ var snapshotScaleCount int
...
 }
 
-func TestMain(t *testing.T) {
-	// If setup fails stop the test
-	if passed := t.Run("setup", setup); !passed {
-		t.FailNow()
-	}
-	// we can either run cluster domain tests or the other tests
-	// the underlying setup of clusters is different
+func TestExtender(t *testing.T) {
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: You can remove these and rename the test methods in the test files to `TestExtender`, etc

Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Done.

**Pull request merged**